tests/integration/inhibit-if-kmods-is-not-supported/test_kmods_not_supported.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+
+
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
+
+from envparse import env
+
+
+@pytest.mark.good_tests
+def test_good_convertion(shell):
+    convertion = shell(
```

Comment:
```suggestion
    conversion = shell(
```
Replace all ~10 occurrences of _convertion_.